manage.py

```diff
@@ -48,17 +48,27 @@ def add_event_identifier():
 
 @manager.command
 def fix_event_and_speaker_images():
-    events = Event.query.filter(Event.original_image_url.isnot(None),
-                                or_(Event.thumbnail_image_url == None, Event.large_image_url == None,
-                                    Event.icon_image_url == None)).all()
+    events = Event.query.filter(
+        Event.original_image_url.isnot(None),
+        or_(
+            Event.thumbnail_image_url == None,
+            Event.large_image_url == None,
+            Event.icon_image_url == None,
+        ),
+    ).all()
     logger.info('Resizing images of %s events...', len(events))
     for event in events:
         logger.info('Resizing Event %s', event.id)
         resize_event_images_task.delay(event.id, event.original_image_url)
 
-    speakers = Speaker.query.filter(Speaker.photo_url.isnot(None),
-                                    or_(Speaker.icon_image_url == None,
-                                        Speaker.small_image_url == None, Speaker.thumbnail_image_url == None)).all()
+    speakers = Speaker.query.filter(
+        Speaker.photo_url.isnot(None),
+        or_(
+            Speaker.icon_image_url == None,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [comparison to None should be 'if cond is None:'](https://app.codacy.com/manual/fossasia/open-event-orga-server/pullRequest?prid=4884175)